### ISSUE TITLE
reverse the date order in the report header 

### ISSUE THREAD

**Issue by llcc** [simonmichael/hledger#2306]
Hi @simonmichael. 

Is it possible to reverse the date order in the report? Displaying the latest account status first could make it more convenient to read in my opinion, especially when the columns extend beyond the screen.

now:
![image](https://github.com/user-attachments/assets/55e62596-82cf-4c8c-9990-b25313f530fd)

wish:
![image](https://github.com/user-attachments/assets/3e9c8542-c3f5-4bf0-b704-8ca169a718f5)

**Comment by simonmichael** [comment]
We don't currently have that feature, a PR would be needed.

Though it's easier to get data reversed if you transpose, eg

    hledger bal assets -M --transpose --pretty=n | tac

**Comment by llcc** [comment]
Doesn't `tac` reverse the order of lines, not columns?

**Comment by simonmichael** [comment]
Yes. `--transpose` converts the columns to rows (and `--pretty=n` removes the inner borders).

**Comment by simonmichael** [comment]
Ie, I'm sharing a possible workaround for seeing the most recent periods first - it's not what you asked for, I know.

**Comment by the-solipsist** [comment]
Adding as a note here: For this workaround, you'd probably want to limit `--depth=1` or `--depth=2`, as otherwise it'll just be a mess.
